satellite/audit/reservoirchore.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"context"
+	"math/rand"
+	"time"
+
+	"go.uber.org/zap"
+
+	"storj.io/storj/internal/sync2"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/satellite/metainfo"
+)
+
+// ReservoirChore populates reservoirs and the audit queue.
+type ReservoirChore struct {
+	log    *zap.Logger
+	config Config
+	rand   *rand.Rand
+	Queue  *Queue
+	Loop   sync2.Cycle
+
+	MetainfoLoop *metainfo.Loop
```

Comment:
This metainfo loop should be mostly internal, so maybe make it private.

Reply:
should `Loop sync2.Cycle` also be unexported?

Reply:
Ideally yes, however, we need all the loops for controlling tests. There's no other reasonable way to expose them either. So, `sync2.Cycle` is the exception.

Reply:
Ah okay that makes sense.